power_grid_model_c/power_grid_model/include/power_grid_model/math_solver/y_bus.hpp

```diff
@@ -432,11 +432,11 @@ template <bool sym> class YBus {
 template class YBus<true>;
 template class YBus<false>;
 
-} // namespace math_model_impl
+} // namespace math_solver
 
-template <bool sym> using YBus = math_model_impl::YBus<sym>;
+using math_solver::YBus;
 
-using YBusStructure = math_model_impl::YBusStructure;
+using math_solver::YBusStructure;
 
```

Comment:
`YBusStructure` is never used outside this file except in the `test_y_bus.cpp` tests. it should not be exposed

```suggestion
```